board/OrangePi-Plus2E/overlay/boot/loader.conf

```diff
@@ -0,0 +1 @@
+fdt_file=orangepi-plus-2e.dtb
```

Comment:
We currently do not support the HDMI controller nor the display engine on H3 in FreeBSD